src/api/index.js

```diff
@@ -26,15 +26,20 @@ import * as markovChain from "../core/algorithm/markovChain";
 import * as credView from "../analysis/credView";
 import * as credData from "../analysis/credData";
 import * as credResult from "../analysis/credResult";
+import * as credGraph from "../core/credrank/credGraph";
 import * as ledger from "../core/ledger/ledger";
 import * as ledgerUtils from "../core/ledger/utils";
 import * as grain from "../core/ledger/grain";
 import * as identity from "../core/identity";
 
 import * as manager from "./ledgerManager";
 import * as storage from "./ledgerStorage";
+import * as credrank from "./credrank";
 
 const api = {
+  api: {
+    credrank,
+  },
```

Comment:
This will eventually contain graph, grain, and potentially even load. I'd like to cannonize these main APIs at the highest level of library abstraction, since they are the primary entry points for our product.